partner_sale_address/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Copyright (C) 2020-Today: Odoo Community Association (OCA)
```

Comment:
There should be no copyright lines in init files and all license lines (in all files) should be https: